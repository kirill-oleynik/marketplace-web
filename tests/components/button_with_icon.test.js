const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const ButtonWithIcon = require('../../app/components/button_with_icon').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <ButtonWithIcon
        text="TestText"
        icon="TestIcon"
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
