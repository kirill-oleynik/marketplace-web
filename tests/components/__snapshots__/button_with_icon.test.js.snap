// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<button
  className="btn-with-icon "
  type="button"
>
  TestText
  <i
    className="icon icon-TestIcon btn-with-icon__icon"
  />
</button>
`;
